src/node_version.h

```diff
@@ -77,6 +77,6 @@
  * an API is broken in the C++ side, including in v8 or
  * other dependencies.
  */
-#define NODE_MODULE_VERSION 58 /* Node.js v8.0.0 */
+#define NODE_MODULE_VERSION 59 /* Node.js v8.0.0 */
```

Comment:
v9.0.0?

Reply:
I've sent a PR to improve comments regarding module version

https://github.com/nodejs/node/pull/16303